cmd/thanos/tools_bucket.go

```diff
@@ -211,6 +212,8 @@ func (tbc *bucketReplicateConfig) registerBucketReplicateFlag(cmd extkingpin.Fla
 
 	cmd.Flag("single-run", "Run replication only one time, then exit.").Default("false").BoolVar(&tbc.singleRun)
 
+	cmd.Flag("concurrency-level", "Number of goroutines to use for replication.").Default("1").IntVar(&tbc.concurrencyLevel)
```

Comment:
let's rename it to `concurrency` or `replicate-concurrency` or something like that. `concurrency-level` is not a common name we use here

Reply:
made the required changes